src/App.tsx

```diff
@@ -1,109 +1,39 @@
 import React from 'react';
 
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import './components/PostInfo/PostInfo.scss';
+import './components/CommentList/CommentList.scss';
+import './components/UserInfo/UserInfo.scss';
+
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getUser(id: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === id);
+
+  return foundUser || null;
```

Comment:
I thought about if this function should return `null` or `undefined` by default. And yes, I like your idea to return `null`. `Post` seems better with `user: null` type